src/main/java/uk/gov/hmcts/darts/dailylist/service/impl/DailyListServiceImpl.java

```diff
@@ -54,6 +65,20 @@ public void processIncomingDailyList(DailyListPostRequest postRequest) {
         }
     }
 
+    @Override
+    @SchedulerLock(name = "DailyListService_Housekeeping",
+        lockAtLeastFor = "PT1M", lockAtMostFor = "PT5M")
+    @Scheduled(cron = "${darts.daily-list.housekeeping.cron}")
+    @Transactional
+    public void runHouseKeeping() {
+        if (housekeepingEnabled) {
```

Comment:
Maybe log if it's not enabled?

Reply:
as discussed, might be better to not do this